prisma/schema.prisma

```diff
@@ -116,6 +169,7 @@ model Project {
   deletedAt DateTime?
 
   team         UserProjects[]
+  organization ProjectOrganization[]
```

Comment:
we don't need the pointer to the organization that owns it (similar to how we don't need the pointer to the user who owns it) the ownership is managed through UserProject & OrganizationProject models

Reply:
actually, we do need it but it should be `ProjectOrganization?` single and optional